javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/reflectionmodel/ReflectionMethodResolutionLogic.java

```diff
@@ -59,22 +59,20 @@
         }
 
         for (ResolvedReferenceType ancestor : scopeType.getAncestors()) {
-            ancestor.getTypeDeclaration().ifPresent(ancestorTypeDeclaration -> {
-                SymbolReference<ResolvedMethodDeclaration> ref = MethodResolutionLogic.solveMethodInType(ancestorTypeDeclaration, name, parameterTypes, staticOnly);
-                if (ref.isSolved()) {
-                    methods.add(ref.getCorrespondingDeclaration());
-                }
-            });
+            ancestor.getTypeDeclaration()
+                    .flatMap(ancestorTypeDeclaration ->
+                            MethodResolutionLogic.solveMethodInType(ancestorTypeDeclaration, name, parameterTypes, staticOnly)
+                                    .getCorrespondingDeclaration()
+                    ).ifPresent(methods::add);
```

Comment:
A very minor stylistic thing -- could you move the calls to `ifPresent` to their own line please?